google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -327,6 +327,10 @@ def bucket bucket_name,
         #   other than the current project, and that project is authorized for
         #   the currently authenticated service account, transit costs will be
         #   billed to the given project. The default is `nil`.
+        # @param [Boolean] autoclass_enabled The bucket's autoclass configuration.
+        #   Buckets can have either StorageClass OLM rules or Autoclass, but
+        #   not both. When Autoclass is enabled on a bucket, adding StorageClass
+        #   OLM rules will result in failure.
```

Comment:
Thanks for the detailed notes. I recommend adding link to the docs as well:

`For more information, see [Autoclass](https://cloud.google.com/storage/docs/autoclass).`

Reply:
Added the link.